CONTRIBUTING.md

```diff
@@ -1,61 +1,92 @@
-<p align="center">
-  <picture>
-    <source srcset="./common/images/Kadena.JS_logo-white.png" media="(prefers-color-scheme: dark)"/>
-    <img src="./common/images/Kadena.JS_logo-black.png" width="200" alt="kadena.js logo" />
-  </picture>
-</p>
-
 # Contributing to Kadena.js
 
-:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+## Table Of Contents
 
-The following is a set of guidelines for contributing to Kadena.js, which is hosted here 
-[Kadena.js](https://github.com/kadena-io/kadena.js) on GitHub.
+- Development
+- Tests
+- Making a Pull Request
+- Conventions
+- Publishing Packages
 
-### Table Of Contents
+Do you want to file a bug? Please [open a new issue][1].
 
-- Code of Conduct
-- Conventions
-- Issues
-- Pull Requests
+## Development
+
+[Install Node.js][2] if you haven't already. Then install Rush:
+
+```bash
+pnpm install --global @microsoft/rush
+```
+
+Clone the repository and install dependencies:
```

Comment:
Just to double check as I'm new, should external contributors just clone or fork the project? If just cloning maybe needed to be mentioned that new feature brunch should be created and PR created to master

Reply:
Good point, external contributors will have to fork, I'll add that.